src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -136,6 +136,8 @@
     <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.xml" />
     <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.$(Platform).xml"
                              Condition="Exists('$(ILLinkDirectory)ILLink.Substitutions.$(Platform).xml')" />
+    <ILLinkSubstitutionsXmls Include="$(ILLinkDirectory)ILLink.Substitutions.iOS.xml"
+                             Condition="'$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'" />
```

Comment:
Is TargetsiOS not covered by the previous entry?

Reply:
Platform in this context is architecture not OS